### PR TITLE
NEWS: mention that we intend to retrigger udev devices on package upg…

### PR DESCRIPTION
**Pull request by poettering**
…rade

Also, mention RISCV GPT partition types have been defined.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -86,6 +86,33 @@ CHANGES WITH 247 in spe:
...
+          is consistent with the updated rule set. This means udev rules must
+          be ready to be retriggered with a "change" action any time, and
+          result in correct and complete udev database entries. While the
+          majority of udev rule files known to us currently get this right,
+          some don't. Specifically, there are udev rules files included in
+          various packages that only set udev properties on the "add" action,
+          but do not handle the "change" action. If a device matching those
+          rules is retriggered with the "change" action (as is intended here)
+          it would suddenly lose the relevant properties. This always has been
+          a problematic, but as soon as all udev devices are triggered on
```
Comment (bluca) [inline NEWS:105]: ```a problematic``` -> ```problematic```

**Comment by mbiebl** [comment]
Unfortunately, running `udevadm trigger -a change` during the upgrade from v246 to v247 doesn't really help. The system breaks in various ways. We can't retroactively fix existing udev rules. So while being able to run `udevadm trigger -a change` during udev updates is a laudable goal in the fture, it doesn't fix #17605

**Comment by keszybz** [comment]
>  So while being able to run `udevadm trigger -a change` during udev updates is a laudable goal in the fture, it doesn't fix #17605

Yeah, this NEWS entry is mostly in preparation if we have a similar issue in the future.

> `a problematic` -> `problematic`

Yeah, the text should be updated a bit. But let's merge and fix later.

File: NEWS (hunk 1 above)
Comment (yuwata) [reply NEWS:105]: #17714.

**Pull request merged**